firefox-ios/Shared/Supporting Files/co.lproj/SuggestedSites.strings

```diff
@@ -0,0 +1,3 @@
+/* Title for X (formerly Twitter) tile in the suggested sites section of the homepage. */
+"SuggestedSites.X.Title.v131" = "X (Twitter)";
```

Comment:
@Delphine Assuming I'm reading this correctly, this string looks a bit odd. None of the other translations put the old name in brackets, nor do I think they should. What can we do to get it updated? 🤔

Reply:
cc @flodolo

Reply:
Since this is already merged, do we need a new key but since this is a one off thing can we change it on the l10n pontoon side and fetch it. @Delphine 🤔

Reply:
This would be a question more for Content Design folks. I don't recall who is working on this part of iOS anymore, so I pinged them in their channel

Reply:
Ok so I'm very sorry for the confusion: I misread the initial ask and thought this was about the original "en-US" string, which failed to be updated to "X".
It's not, it's the translation from "co" locale, so I can get in touch with the localizer to update. 
Apologies for the noise